docs/source/array-creation.rst

```diff
@@ -352,6 +352,45 @@ Files may be retrieved by running `da.from_tiledb` with the same URI, and any
 necessary arguments.
 
 
+Intermediate storage
+--------------------
+
+.. autosummary::
+   store
+
+In some cases, one may wish to store an intermediate result in long term
+storage. This differs from ``persist``, which is mainly used to manage
+intermediate results within Dask that don't necessarily have longevity.
+Also it differs from storing final results as these mark the end of the Dask
+graph. So lack anyway to be reused in other graphs without reloading the data.
```

Comment:
This feels like a fragment of a sentence rather than a full sentence.  Maybe combine it somehow with the previous sentence or give it it's own start?

Reply:
Good catch. Have reworded.